src/ampycloud/wmo.py

```diff
@@ -169,17 +176,19 @@ def alt2code(val: Union[int, float]) -> str:
     """ Function that converts a given altitude in hundreds of ft (3 digit number),
     e.g. 5000 ft -> 050, 500 ft -> 005.
 
-    Below 10'000 ft, the value is floored to the nearest 100 ft. Above 10'000 ft, the value is
-    floored to the nearest 1000 ft.
-
-    Reference: *Aerodrome Reports and Forecasts, A Users' Handbook to the Codes*,
-    WMO-No.782, 2020 edition. https://library.wmo.int/?lvl=notice_display&id=716
-
     Args:
         val (int, float): the altitude to convert, in feet.
 
     Returns:
         str: the corresponding METAR code chunk
+
+    Below 10'000 ft, the value is floored to the nearest 100 ft. Above 10'000 ft, the value is
```

Comment:
How do we handle the variable resolution of the Met Report?
See GM1 MET.TR.200(a): 0-250 ft --> 50 ft; 300-10000 ft --> 100 ft
The alt2code function can only be used for AutoMETAR formatting. 
For local Met Report we need to pass the unconverted altitudes, which I assume is already done in autometpy.
We could add a comment to this function.

Reply:
If my memory serves me well, the autometpy calculator directly extracts the layer altitudes, NOT any WMO/ICAO-formatted codes. @nestabur would know for certain. I believe this was done precisely to keep all the operationally-relevant regulations-related formatting handled by a single software (i.e. SMART).

I will add a warning to this docstring to mention that the routine does not (currently) allow to implement EASA's rule AMC1 MET.TR.205(e)(3) and set the resolution to 50ft up to 300 ft for aerodromes with established low-visibility procedures.